druid/src/widget/textbox.rs

```diff
@@ -336,8 +336,10 @@ impl Widget<String> for TextBox {
 
     fn lifecycle(&mut self, ctx: &mut LifeCycleCtx, event: &LifeCycle, _data: &String, _env: &Env) {
         match event {
-            LifeCycle::WidgetAdded => ctx.invalidate(),
-            LifeCycle::Register => ctx.register_for_focus(),
+            LifeCycle::WidgetAdded => {
+                ctx.invalidate();
```

Comment:
I don't think we need this. If removing it causes behavior changes, we should probably understand why. That said, it's probably not doing any harm.